core/src/main/java/org/elasticsearch/rest/action/support/RestActions.java

```diff
@@ -64,18 +73,24 @@ public static long parseVersion(RestRequest request, long defaultVersion) {
     }
 
     static final class Fields {
+        static final String _NODES = "_nodes";
```

Comment:
I'm still not following you? What's wrong with the `static` reference? Whether you use a constant string field (`static final String` assigned from a literal string) or just use literal strings in all the places that you would use the constant, the effect is the same: a single `String` is put in the constant pool and that constant is pushed onto the stack when needed using `ldc`. The java compiler effectively interns all literal strings, and this is the same effect as using a constant string field.